lib/ansible/modules/packaging/language/pip.py

```diff
@@ -529,10 +529,10 @@ def __str__(self):
 
 def main():
     state_map = dict(
-        present=['install'],
-        absent=['uninstall', '-y'],
-        latest=['install', '-U'],
-        forcereinstall=['install', '-U', '--force-reinstall'],
+        present='install',
```

Comment:
Why did you revert this to string?

Reply:
When user input extra option like `--user --root xxxxxx`, the list type cannot handle it well.

Reply:
well, it can always be str.split